### PR TITLE
Update new_organization.rst

### PR DESCRIPTION
**Pull request by gurjot03**
updated the link to reference the document _doc_adminguide_hosting (docs/adminguide/hosting/index.rst) in 
 docs/userguide/new_organization.rst
#5649

### CODE REVIEW

File: docs/userguide/new_organization.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Otherwise, you can click on `Create an organization` in the Menu of the login sc
 .. note::
 
     Another possibility (though more technical) is to host you own Parsec server and create the organization on it using the Parsec CLI. You can then join that organization using an organization bootstrap URL.
-    You can read about this in the `the Hosting server <https://github.com/Scille/parsec-cloud/blob/master/docs/HOSTING.md>`_ part.
+    You can read about this in the `the Hosting server <https://github.com/Scille/parsec-cloud/blob/master/docs/adminguide/hosting/index.rst>`_ part.
```
Comment (FirelightFlagboy) [inline docs/userguide/new_organization.rst:27]: In RST you can use [`cross-referencing`](https://www.sphinx-doc.org/en/master/usage/referencing.html#ref-role).

The document we want to reference here already has a label

https://github.com/gurjot03/parsec-cloud/blob/e160ea69d5e966754dd680043d316e5ae83a4190/docs/adminguide/hosting/index.rst?plain=1#L5

So you can do:

```suggestion
    You can read about this in the :ref:`the Hosting server <_doc_adminguide_hosting>` part.
```

Comment (gurjot03) [reply docs/userguide/new_organization.rst]: please take a look now

Comment (FirelightFlagboy) [reply docs/userguide/new_organization.rst]: Yes, almost done https://github.com/Scille/parsec-cloud/pull/5656#discussion_r1375907014

File: docs/userguide/new_organization.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ Otherwise, you can click on `Create an organization` in the Menu of the login sc
 .. note::
 
     Another possibility (though more technical) is to host you own Parsec server and create the organization on it using the Parsec CLI. You can then join that organization using an organization bootstrap URL.
-    You can read about this in the `the Hosting server <https://github.com/Scille/parsec-cloud/blob/master/docs/HOSTING.md>`_ part.
+    You can read about this in the :ref:`the Hosting server <_doc_adminguide_hosting>` part.
```
[the remark this answers is not in the document]
Comment (gurjot03) [reply docs/userguide/new_organization.rst]: removed "_"

**Review by FirelightFlagboy — APPROVED** [review]

**Comment by gurjot03** [comment]
please merge

**Comment by FirelightFlagboy** [comment]
Before merging your PR, we need to update the translation with:

```
make intl-fr
```

I can do it on my side but not now

**Pull request merged**